.travis.yml

```diff
@@ -1,11 +1,15 @@
-sudo: false
```

Comment:
Removing this as it's deprecated according [to Travis documentation](https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration#timeline---its-happening-fast).